MODULE.bazel

```diff
@@ -17,7 +17,7 @@ bazel_dep(
     repo_name = "io_bazel_stardoc",
 )
 
-apple_cc_configure = use_extension("//crosstool:setup.bzl", "apple_cc_configure_extension")
+apple_cc_configure = use_extension("//crosstool:apple_cc_configure_extension.bzl", "apple_cc_configure_extension")
```

Comment:
Is this file rename a breaking change for users? We have a blurb in the README that references the old name.

Reply:
It would be, yes. I think it's better to wait for apple_support to drop Bazel 5 support and then land this commit without the need for this breaking change.